java/java.completion/test/unit/data/goldenfiles/org/netbeans/modules/java/completion/JavaCompletionTaskTest/10/override.pass

```diff
@@ -0,0 +1 @@
+Override
```

Comment:
I don't think we need the golden files that are the same for 1.8 and 10 and other versions, right? If the tests don't find the golden output for e.g. version 10, they should look for the golden output of older versions. I.e. it should be enough to specify a golden output for 1.8, and only re-specify it if it changes in a newer version.

Reply:
Had reduced the golden files in last commit.